specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/jobs.json

```
@@ -0,0 +1,539 @@
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CodeJobIdParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "CodeJob definition object.",
+            "schema": {
+              "$ref": "#/definitions/CodeJobCreateResourceArmDto"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated CodeJob",
+            "schema": {
+              "$ref": "#/definitions/CodeJobResourceArmDto"
+            }
+          },
+          "201": {
+            "description": "Successfully created CodeJob",
+            "schema": {
+              "$ref": "#/definitions/CodeJobResourceArmDto"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "CodeJob"
+        ],
+        "description": "Gets a CodeJob by name/id.",
+        "operationId": "CodeJob_Get",
+        "x-ms-examples": {
+          "Get CodeJob": {
+            "$ref": "./examples/getCodeJob.json"
+          }
+        },
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CodeJobIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CodeJobResourceArmDto"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/codeJobs": {
+      "get": {
+        "tags": [
+          "CodeJob"
+        ],
+        "description": "Lists CodeJobs in the workspace.",
+        "operationId": "CodeJob_List",
```

Suggestion: usually operationId is in this pattern: `PuralNoun_Verb`, could we change this to 
```suggestion
        "operationId": "CodeJobs_List",
```